src/php/Compiler/Emitter/OutputEmitter/LiteralEmitter.php

```diff
@@ -186,4 +188,22 @@ private function emitVector(PersistentVector $x): void
         }
         $this->outputEmitter->emitStr('])', $x->getStartLocation());
     }
+
+    private function emitArray(array $x): void
+    {
+        $this->outputEmitter->emitStr('[');
+        $count = count($x);
+        $index = 0;
+        foreach ($x as $key => $value) {
+            $this->outputEmitter->emitLiteral($key);
+            $this->outputEmitter->emitStr(' => ');
+            $this->outputEmitter->emitLiteral($value);
+            if ($index < $count - 1) {
+                $this->outputEmitter->emitStr(', ');
+            }
```

Comment:
I wonder if this condition is needed? I mean, you can have a comma after the last element in an array in PHP. Or is this for another reason?

Reply:
Well, I don't think this is a big deal anyway. This is pure cosmetics to make the output PHP code nicer. And considering we have a compiler, there is no downtime performance or anything similar. This is totally fine.
<img width="1248" alt="Screenshot 2022-03-03 at 08 08 23" src="https://user-images.githubusercontent.com/5256287/156514227-9747c26f-0ac4-4cd4-914e-bc0c8697830c.png">